### PR TITLE
Weekend/homework

### PR DESCRIPTION
**Pull request by lenzloesche**
# things I have done:
- I added a README.md
- I made it responsive.

I don't know what else to do with this website. :)

### CODE REVIEW

**Comment by sahedw** [comment]
just downloaded the files and looks good! Nice job :)

**Review by nikokreissl — APPROVED** [review]
Nothing to add from my side, good job. :)

**Review by actyralf — CHANGES_REQUESTED** [review]
Hey Lenz,

great job!

I have some comments regarding the initial commit - I can't do this line by line because the affected lines are not part of the pull request.

Here they are:

- Consider to use a `ul` with `li`s for the navbar for accessibility reasons 
- Consider to use `<section>` tags instead of `<div>` where feasible
- Make sure that all your images have an appropriate `alt` attribute
- Make sure your inputs have corresponding labels: You have a label for your e-mail field, but it's not linked to the input
- JS: `calculate` is not a good name for the variable holding the progress percentage. You should use verbs only for functions. Consider to use sth. like `progressPercentage` or similar.
- JS: You should link the `.js` file with the `defer` attribute to ensure that HTML/CSS is completely loaded before JS gets executed.
- We want to develop mobile first, so please do your default layout for mobile and your media queries for `min-width` instead of `max-width`
- Don't use `<a>` tags with empty `href` attribute
- You used `<h3>` tags to wrap you nav items. Consider selecting the nav items with an appropriate CSS selector and style them via this selector. `<h3>` should be reserved for headings.

File: index.html [hunk 1]
```diff
@@ -6,128 +6,133 @@
...
 
-        <h3 class="navelement"><a href="#links">Links</a> -</h3>
+        <li class="nav__element"><a href="#links">Links</a> -</li>
 
-        <h3 class="navelement"><a href="#kontakt">Contact</a></h3>
+        <li class="nav__element"><a href="#kontakt">Contact</a></li>
 
-        <div class="arrow" id="arrow">
-          <img class="arrowimg" src="bilder/arrow.gif" alt="" />
+        <div class="nav__arrow" id="arrow">
```
Comment (actyralf) [inline index.html]: You should have only `<li>` tags inside your list. Please wrap the `<div>` inside an `<li>`

File: index.html [hunk 2]
```diff
@@ -6,128 +6,133 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./styles.css" />
     <title>Lenz Lösches Homepage</title>
-    <script src="script.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body id="body">
     <header></header>
```
Comment (actyralf) [inline index.html]: `<header>` is empty - you could wrap your nav inside it...

File: index.html [hunk 3]
```diff
@@ -6,128 +6,133 @@
...
-        <div class="skillsgrid">
-          <div class="skillitem"><p>HTML</p></div>
-          <div class="skillitem"><p>CSS</p></div>
-          <div class="skillitem"><p>JavaScript</p></div>
-          <div class="skillitem"><p>React</p></div>
-          <div class="skillitem"><p>Git</p></div>
-          <div class="skillitem"><p>Github</p></div>
-          <div class="skillitem"><p>CleanCode</p></div>
+      <section class="containerdiv background-skills" id="skills">
+        <div class="skill">
```
Comment (actyralf) [inline index.html]: semantically this looks like a list - please consider to use `<ul>` instead of `<div>`

File: index.html [hunk 4]
```diff
@@ -6,128 +6,133 @@
...
-      </div>
+      </section>
       <div class="line"></div>
 
-      <div class="containerdiv background-vita" align="center" id="lebenslauf">
-        <div class="vitatable">
-          <div class="about">
+      <section
+        class="containerdiv background-vita"
+        align="center"
```
Comment (actyralf) [inline index.html]: The `align` attribute is deprecated (https://www.w3.org/TR/html401/present/graphics.html).
Please use the CSS `text-align` property to achieve the effect.

**Review by actyralf — APPROVED** [review]
Hey Lenz,

great job! I left a few tiny comments - feel free to merge anyhow and maybe consider them for the next PR. Alternatively you can fix them right now but you don't have to request another review for the fix.

**Pull request merged**